util/src/main/scala/geotrellis/util/StreamBytes.scala

```diff
@@ -0,0 +1,22 @@
+package geotrellis.util
+
+import java.io.InputStream
+import java.nio.ByteBuffer
+
+abstract class StreamBytes(chunkSize: Int) {
```

Comment:
Yeah, I realized that after you pointed that out yesterday. I'll be adding Docstrings for sure to all of my code.